backend/src/bundles/hard-skills/hard-skills.model.ts

```diff
@@ -19,7 +18,7 @@ class HardSkillsModel extends AbstractModel {
     public static override relationMappings = {
         talents: {
             relation: Model.ManyToManyRelation,
-            modelClass: UserDetailsModel,
+            modelClass: '',
```

Comment:
why do we need these changes?

Reply:
Described earlier. Something with path or something else.
https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/pull/190#discussion_r1314600310